src/importData.jl

```diff
@@ -54,6 +54,141 @@ function import_combination(filename::String)
     return gs[:, :, 2:25], gs[:, :, 27:50], perc, total
 end
 
+""" This function imports the new round of data and filters. """
+function import_data()
+    # 96 conditions(not separating replicates) with 191 data points,
+    # and 1158 conditions(not separating conditions) with 192 data points.
+    # 18336 / 191 = 96
+    # 222336 / 193 = 1152
+    # so we have a total of 1254 conditions
+    basePath = joinpath(dirname(pathof(DrugResponseModel)), "..", "data")
+
+    d = zeros(193, 1248, 2) # time points x conditions x G1% and total
+
+    dataname = "AU565_round2and3_model_subset_level2.csv"
+    data = readdlm(joinpath(basePath, dataname), ',')
+
+    conditions = Vector{String}()
+
+    for i = 1:96 # first set that have 191 data points
+        d[1:191, i, 1] = data[(191 *(i-1)) + 2 : (191 * i) + 1, 9]
+        d[1:191, i, 2] = data[(191 *(i-1)) + 2 : (191 * i) + 1, 8]
+        push!(conditions, data[(191 *(i-1)) + 2, 5])
+    end
+    for i=96:1247 # the rest that have 193 data points
+        d[:, i+1, 1] = data[(193 *(i-1)) + 3 : (193 * i) + 2, 9]
+        d[:, i+1, 2] = data[(193 *(i-1)) + 3 : (193 * i) + 2, 8]
+        push!(conditions, data[(193 *(i-1)) + 3, 5])
+    end
+
+    # filter
+    for i = 1:size(d, 2)
+        d[:, i, 1] = savitzky_golay_filter(d[:, i, 1], 41, 3)
```

Comment:
I'd suggest having a function that takes in a vector and does this filtering so that the filter settings are defined in one place.